### ISSUE TITLE
ILP Data Handling on Date

### ISSUE THREAD

**Issue by kaon** [questdb/questdb#1938]
### Describe the bug

Ingest following data using Python 3 and ILP protocol on database version 6.2.1 and version 6.2 respectively:

` crypto_funding_rate,exchange=FTX,contract=ICX rate=-1.8e-05,time=2022-03-09T08:00:00+00:00\n`

DB (v6.2.1) : nothing will be created
DB (v6.2) : Expected data can be queried 

No error was found in the log as well.

### To reproduce

Using following python code:

```
sock.connect((HOST, PORT))
sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
sock.sendall('crypto_funding_rate,exchange=FTX,contract=ICX rate=-1.8e-05,time=2022-03-09T08:00:00+00:00\n'.encode())
sock.close()

```

### Expected Behavior

Record will be inserted into database.

### Environment

```markdown
- **QuestDB version**: 6.2.1
- **OS**: Ubuntu 20.04 LTS
- **Browser**: Firefox 97.0.1 (64 bit)
```

**Comment by sebfia** [comment]
Similar problem here - had to switch back to 6.2

**Comment by puzpuzpuz** [comment]
ILP transactions are implicit, so it may take some time for your rows to be committed. This documentation page explains commit strategies in a good level of detail:
https://questdb.io/docs/reference/api/ilp/tcp-receiver#commit-strategy

If you're only inserting a single row, either idle table timeout or interval-based commit should kick in. The first one should happen after `line.tcp.min.idle.ms.before.writer.release` (10 sec by default). The second one happens after the commit interval (150 sec by default).

@kaon your message get rejected since `time` field's value should be an Epoch timestamp in microseconds, not a formatted string:
https://questdb.io/docs/reference/api/ilp/columnset-types#timestamp

The only thing we could improve is adding some error logging, so that the database wouldn't be rejecting the row in total silence.
@glasstiger WDYT?

**Comment by puzpuzpuz** [comment]
> Similar problem here - had to switch back to 6.2

@sebfia please the above comment. You should use the latest version (6.2.1) since it contains a number of bugfixes and optimizations.

**Comment by kaon** [comment]
@puzpuzpuz thanks for your sharing. It may be a bad practice, what can we do if we wanna store such time in formatted string? 

From https://questdb.io/docs/reference/api/ilp/overview, 
> When the table_name does not correspond to an existing table, QuestDB will create the table on the fly using the name provided. Column types will be automatically recognized and assigned based on the data.

QDB should be able to put the data to default column type instead of discarding them in silence.

**Comment by puzpuzpuz** [comment]
> It may be a bad practice, what can we do if we wanna store such time in formatted string?

The thing is that your timestamps will be stored as epoch timestamps in any case. It's only a matter of an intermediate representation. It should be straightforward to convert your timestamp/date objects in the client application to an epoch timestamp instead of a formatted string. If you still think that having support for formatted timestamp string, please open a new issue for that enhancement.

> QDB should be able to put the data to default column type instead of discarding them in silence.

The documentation describes the behavior in case of a valid row. As for the invalid row case, the row gets discarded as a whole. I agree that the database should print some warn/error messages to logs for problematic rows - I'll take a look at this.

**Comment by puzpuzpuz** [comment]
> I agree that the database should print some warn/error messages to logs for problematic rows - I'll take a look at this.

Should be fixed in #2085
